test/testtokenize.cpp

```diff
@@ -8108,6 +8108,9 @@ class TestTokenizer : public TestFixture {
 
         // Ticket #9664
         ASSERT_NO_THROW(tokenizeAndStringify("S s = { .x { 2 }, .y[0] { 3 } };"));
+
+        ASSERT_THROW_EQUALS(tokenizeAndStringify("void f() { assert(a==()); }"), InternalError, "syntax error: == (");
```

Comment:
hmm `== (` is not a syntax error that can be seen in valid code. Could you try to write `==()` instead?

Reply:
`== (` should not produce syntax errors, since I used `Token::simpleMatch(tok->next(), "( )")` to match this case

Reply:
yes but the user can not know what pattern you use. I'd prefer that the message say `==()`